src/NumericalAlgorithms/FiniteDifference/MonotonicityPreserving5.hpp

```diff
@@ -114,18 +118,18 @@ struct MonotonicityPreserving5Reconstructor {
  *  q^\text{MP} = q_j + \text{minmod}(q_{j+1} - q_j, \alpha(q_j - q_{j-1}))
  * \f}
  *
- * for a given value of \f$\alpha\f$ which is usually set to \f$\alpha=4\f$.
+ * for a given value of \f$\alpha\f$, which is usually set to \f$\alpha=4\f$.
  *
  * If \f$ (q_{j+1/2}^\text{OR} - q_j)(q_{j+1/2}^\text{OR} - q^\text{MP}) \leq
- * \epsilon \f$ where \f$\epsilon\f$ is a small tolerance value, use $q_{1/2}
+ * 0\f$ where \f$\epsilon\f$ is a small tolerance value, use $q_{1/2}
```

Comment:
Why change `epsilon` -> `0` and then say what `epsilon` is? I would say leave the previous sentence, but add `value (discussed below)`